datafoundation/templates_storagecluster.go

```diff
@@ -96,36 +94,48 @@ var StorageClusterTemplate = ocsv1.StorageCluster{
 			"mon":            GetResourceRequirements("mon"),
 			"crashcollector": GetResourceRequirements("crashcollector"),
 		},
+		BackingStorageClasses: []ocsv1.BackingStorageClass{{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: backingStorageClassName,
+			},
+			Provisioner: "ebs.csi.aws.com",
+			Parameters: map[string]string{
+				"iops":       "12000",
+				"throughput": "250",
+			},
+		}},
 		StorageProfiles: []ocsv1.StorageProfile{{
 			DeviceClass: "ssd",
-			Name:        "default",
+			Name:        "default-storage-profile",
 			BlockPoolConfiguration: ocsv1.BlockPoolConfigurationSpec{
 				Parameters: map[string]string{
-					"pg_autoscale_mode": "on",
+					"pg_autoscale_mode": "off",
 					"pg_num":            "128",
 					"pgp_num":           "128",
+					"target_size_ratio": ".49",
 				},
 			},
 			SharedFilesystemConfiguration: ocsv1.SharedFilesystemConfigurationSpec{
 				Parameters: map[string]string{
-					"pg_autoscale_mode": "on",
-					"pg_num":            "128",
-					"pgp_num":           "128",
+					"pg_autoscale_mode": "off",
+					"pg_num":            "512",
+					"pgp_num":           "512",
```

Comment:
@bindrad this seems wrong. In a previous version of ODF, the pg and pgp allocation was a calculation in code. I don't think this setup matches what we had there as we know we had a resource constraint. 

@ezio-auditore Can you help with this as the original calculation (in ocs-operator) was your contribution?

Reply:
I did discuss the configuration with @ezio-auditore  and got the confirmation that this is the one being used in the product.